src/components/TodoInfo/TodoInfo.spec.tsx

```diff
@@ -3,40 +3,30 @@ import { TodoInfo } from './TodoInfo';
 
 describe('TodoInfo', () => {
   it('should have a correct title', () => {
-    const user1 = {
-      id: 1,
-      name: 'Leanne Graham',
-      username: 'Bret',
-      email: 'Sincere@april.biz',
+    const user1Todo = {
+      userName: 'Leanne Graham',
+      userEmail: 'Sincere@april.biz',
+      todoPost: {
+        id: 1,
+        title: 'Learn HTML',
+        completed: true,
+      },
```

Comment:
Don't change or remove tests

Reply:
so, then I have to remake this project( ehh